.evergreen/.evg.yml

```diff
@@ -861,11 +861,35 @@ tasks:
           vars:
             file: ".evergreen/static-checks.sh"
 
-    - name: "test"
+    - name: "test-bson-and-crypt"
+      commands:
+        - func: "run tests"
+          vars:
+            TESTS: 'bson:test bson-kotlin:test bson-kotlinx:test bson-scala:test bson-record-codec:test mongodb-crypt:test'
```

Comment:
These run in seconds, but separating them into their own task has benefits anyway.  This way all these tests don't pollute the logs of the per-MongoDB test runs with thousands of irrelevant tests.

Reply:
@rozza not sure about including `bson-kotlin/bson-kotlinx` here.  Looking a thet `kotlin-tests` task, I see `bson-kotlin` tests included, but not `bson-kotlinx`.

Reply:
Not needed - so all good:

```
> ./gradlew kotlinCheck --dry-run -i | grep "\(:test \|:integrationTest \)"
:bson-kotlin:test SKIPPED
:bson-kotlinx:test SKIPPED
:driver-kotlin-coroutine:test SKIPPED
:driver-kotlin-coroutine:integrationTest SKIPPED
:driver-kotlin-sync:test SKIPPED
:driver-kotlin-sync:integrationTest SKIPPED
```

Reply:
Just to confirm: I should change this to:

```
TESTS: 'bson:test bson-record-codec:test mongodb-crypt:test'
```

because Kotlin and Scala bson tests already run in other variants?

Reply:
Correct - looks like you already have changed it :)

Reply:
So I have, Ross, so I have...